ex01.exs

```diff
@@ -27,12 +27,28 @@ defmodule Ex01 do
           blank lines, vertical alignment
   """
   
-  def counter(value \\ 0) do
-  end
-
+    def counter(value \\ 0) do
```

Comment:
Good, but please keep an eye on consistent layout. Your use of whitespace is a little wild :)